environments/overlays/cicd/pipeline-bot.yaml

```
@@ -0,0 +1,22 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: build-bot
```

Maybe we can give this a more meaningful name. What about
```suggestion
  name: pipeline-credentials-sa
```

---

Fully agree. Taking note that all PipelineRuns must be changed in terms of Service Account to that new name.